libs/community/langchain_community/tools/ddg_search/tool.py

```diff
@@ -62,8 +64,9 @@ def _run(
     ) -> str:
         """Use the tool."""
         res = self.api_wrapper.results(query, self.max_results, source=self.backend)
-        res_strs = [", ".join([f"{k}: {v}" for k, v in d.items()]) for d in res]
-        return ", ".join([f"[{rs}]" for rs in res_strs])
+        res_strs = [", ".join([f"{k}: {v}" for k, v in d.items() 
+                    if not self.keys_to_include or k in self.keys_to_include]) for d in res]
+        return self.results_separator.join([f"[{rs}]" for rs in res_strs])
```

Comment:
should we replace `f"[{rs}]"` with just `rs`? not obviouus to me former should be hardcoded, especially if separator is configurable

Reply:
Can I just replace with .join(res_strs) then?